tests/e2e/specs/frontend/legacy-template-blocks.test.js

```diff
@@ -74,7 +74,7 @@ describe( 'Legacy Template blocks', () => {
 		} );
 	} );
 
-	describe( 'Product Category block', () => {
+	describe( 'Product by Category block', () => {
```

Comment:
Heads-up that the name of these blocks is going to be updated in https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5890, but in any case and if you want to fix it, I think there is a missing `s` here: :slightly_smiling_face: 
```suggestion
	describe( 'Products by Category block', () => {
```